### DISCUSSION TITLE
Use two scripts use two scripts simultaneously

### DISCUSSION THREAD

**Discussion by tafo86awtech** [emqx/MQTTX#920] [Ideas]
Hi! I think is would be very useful the possibily of use one script for the published messages and one for the received messages.
At moment i can choice only one script at the time, but often i have to use one script for published ones  and onether for received ones.

One example is a code/decode message situation, when I publish I need to code, when I receive I need to decode

**Comment by ysfscream** [comment]
YES. Looks better! I'll think about implementing it, but it may take some time these days, so I'll turn it into an issue to track the implementation of this feature.

**Reply by ysfscream** [reply]
It's here: #921
